editor/store/effects.js

```diff
@@ -175,6 +176,14 @@ export default {
 			) );
 		}
 
+		// The server can return a sanitised version of the slug,
+		// in which case we need to update our local copy.
+		if ( get( edits, 'slug' ) !== post.slug ) {
```

Comment:
When will `edits.slug` throw? FYI, passing a string as the second argument of `get` (or any Lodash function which accepts a "path") will invoke a very computationally-intensive parse sequence, specifically for use like `get( obj, 'deep.object.property.value' )`. Typically recommend always passing array as the second argument of `get`, even if only one level deep.

Reply:
`edits.slug` will throw if you change the postname, then change it back to the original postname.

I've updated it to be an array, though.